### PR TITLE
vedic 2.0.6 (new formula)

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

basically [this comment](https://github.com/Homebrew/homebrew-core/pull/172593#discussion_r1612443768) to help build from source.

also fixed the carriage issue
```
Formula/v/vedic.rb:1:1: C: Layout/EndOfLine: Carriage return character detected.
class Vedic < Formula ...
^^^^^^^^^^^^^^^^^^^^^
```

closes #172593

### CODE REVIEW

File: Formula/v/vedic.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class Vedic < Formula
+  desc "Simple Sanskrit programming language"
+  homepage "https://vedic-lang.github.io/"
+  url "https://github.com/vedic-lang/vedic/archive/refs/tags/v2.0.6.tar.gz"
+  sha256 "5453386fcebfa48a5f3d39bc832d19edaaaa98d8d36841428ff6a20db0dd0151"
```
Comment (chenrui333) [inline Formula/v/vedic.rb:5]: sha got changed due to re-tagged.
```suggestion
  sha256 "9bfb68dfa8a79c02d52905eb1403267209dae80ad05287b7f3706f14071c4800"
```

**Review by ZhongRuoyu — APPROVED** [review]

**Review by p-linnane — APPROVED** [review]
